cypress/e2e/example.cy.ts

```diff
@@ -3,6 +3,6 @@
 describe('My First Test', () => {
   it('visits the app root url', () => {
     cy.visit('/');
-    cy.contains('h3', 'Documentation');
+    cy.contains('div', 'Counter-Strike');
```

Comment:
I know this is just an example app where the data isn't changing, but when you're testing data like this you'd want to test that something generic exists like a class name for example, rather the content that the data is producing since this will be different most of the time :)